src/CabanaPD_ForceModels.hpp

```diff
@@ -173,7 +175,18 @@ struct BaseTemperatureModel<TemperatureDependent, TemperatureType>
                           const double _temp0 )
         : alpha( _alpha )
         , temp0( _temp0 )
-        , temperature( _temp ){};
+        , temperature( _temp )
+    {
+    }
+
+    // FIXME: use the first model temperature for now.
+    template <typename ModelType1, typename ModelType2>
+    BaseTemperatureModel( const ModelType1& model1, const ModelType2& model2 )
+    {
+        temperature = model1.temperature;
+        alpha = ( model1.alpha + model2.alpha ) / 2.0;
+        temp0 = ( model1.temp0 + model2.temp0 ) / 2.0;
```

Comment:
I don't think this should be averaged

Reply:
Could you explain your reasoning here? I could imagine a simulation where different materials have different temperatures at which they are relaxed (no forces). Maybe even for a triso particle